gallery/psget/psget_get_psget_module.md

```diff
@@ -0,0 +1,61 @@
+---
+description:  
+manager:  kriscv
+ms.topic:  article
+author:  tylerl0706
+ms.prod:  powershell
+keywords:  powershell,cmdlet,gallery
+ms.date:  2017-6-3
+contributor:  tylerl0706
+title:  Get PowerShellGet Module
+ms.technology:  powershell
+---
+
+Get PowerShellGet Module
+========================
+
+### PowerShellGet is an in-box module in the following releases
+- [Windows 10](https://www.microsoft.com/en-us/windows/get-windows-10) or newer
+- [Windows Server 2016](https://technet.microsoft.com/en-us/windows-server-docs/get-started/windows-server-2016) or newer
+- [Windows Management Framework (WMF) 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=50395) or newer
+- [PowerShell 6.0.0-Alpha](https://github.com/PowerShell/PowerShell/releases)
+
+### Get PowerShellGet module for PowerShell versions 3.0 and 4.0
+- [PackageManagement MSI](http://go.microsoft.com/fwlink/?LinkID=746217&clcid=0x409) 
+
+### Get the latest version from PowerShell Gallery
+
+- Before updating PowerShellGet, you should always install the latest Nuget provider. To do that, run the following in an elevated PowerShell session.
+```powershell
+Install-PackageProvider Nuget –Force
+Exit
```

Comment:
You can't copy the dlls into PackageManagement folder as existing dlls are already loaded into the process. That's why exit is required after save-module in PS3/4.